fellow-submissions/Anne-Hyacinthe.md

```diff
@@ -0,0 +1,98 @@
+# Problem Set 2.4 - Scope
+
+**1. How do variables declared with `let`, `const`, and `var` differ? Be sure to touch on _scope_, _reassignment_, and _hoisting_ for each.**
+
+**2. What will the following code log? Why?**
+    ```javascript
+    let a = 'outer';
+
+    function testScope() {
+      let a = 'inner';
+      console.log(a);
+    }
+
+    console.log(a);
+    testScope();
+    console.log(a);
+    ```
+*```outer inner outer``` The first and last log were 'outer' because they don't have a local scope with an 'a' variable so they had to call the value from the global scope.
```

Comment:
Note: Only use one back tick if you are writing "in-line" code snippets. You use three backticks if you are including a block of code.

Okay... a slight revision:
```
outer
inner
outer
``` 
> The first and last log ~~were~~ `"outer"` because they don't have **access to the block scoped variable `a` declared in the body of the `testScope` function.**